RecoTracker/TkSeedGenerator/plugins/BuildFile.xml

```diff
@@ -5,6 +5,42 @@
 <use name="RecoPixelVertexing/PixelTriplets"/>
 <use name="RecoPixelVertexing/PixelTrackFitting"/>
 <use name="RecoPixelVertexing/PixelLowPtUtilities"/>
+<use name="CommonTools/Utils"/>
+<use name="DataFormats/BeamSpot"/>
+<use name="DataFormats/Common"/>
+<use name="DataFormats/Math"/>
+<use name="DataFormats/SiPixelDetId"/>
+<use name="DataFormats/SiStripDetId"/>
+<use name="DataFormats/TrackReco"/>
+<use name="DataFormats/TrackerCommon"/>
+<use name="DataFormats/TrackingRecHit"/>
+<use name="DataFormats/TrajectorySeed"/>
+<use name="DataFormats/TrajectoryState"/>
+<use name="DataFormats/VertexReco"/>
+<use name="FWCore/Framework"/>
+<use name="FWCore/MessageLogger"/>
+<use name="FWCore/ParameterSet"/>
+<use name="FWCore/PluginManager"/>
+<use name="FWCore/Utilities"/>
+<use name="Geometry/CommonDetUnit"/>
+<use name="Geometry/Records"/>
+<use name="Geometry/TrackerGeometryBuilder"/>
+<use name="MagneticField/Engine"/>
+<use name="MagneticField/Records"/>
+<use name="MagneticField/UniformEngine"/>
+<use name="RecoTracker/MeasurementDet"/>
+<use name="RecoTracker/Record"/>
+<use name="RecoTracker/SpecialSeedGenerators"/>
+<use name="RecoTracker/TkHitPairs"/>
+<use name="RecoTracker/TkSeedingLayers"/>
+<use name="RecoTracker/TransientTrackingRecHit"/>
+<use name="TrackingTools/GeomPropagators"/>
+<use name="TrackingTools/KalmanUpdators"/>
+<use name="TrackingTools/MeasurementDet"/>
+<use name="TrackingTools/PatternTools"/>
+<use name="TrackingTools/Records"/>
+<use name="TrackingTools/TrajectoryState"/>
+<use name="TrackingTools/TransientTrackingRecHit"/>
```

Comment:
many of these dependencies are declared in RecoTracker/TkSeedGenerator.
Is it time to cleanup that package dependencies?

Reply:
its correct to have dependencies in both src and plugins BuildFiles assuming both src and plugins use them directly (at least that is my understanding of the correct way to manage BuildFiles).